Rakefile

```diff
@@ -34,7 +34,7 @@ namespace :lint do
     Dir.chdir(apps_dir) do
       HipChat.log 'Linting <b>apps</b> JavaScript...'
       # lint all js/jsx files in dashboardd/app/assets/javascript
-      RakeUtils.system 'grunt jshint:files --glob "../dashboard/app/**/*.js*(x)"'
+      RakeUtils.system './node_modules/.bin/eslint -c .eslintrc.js ../dashboard/app/ --ext .js,.jsx'
```

Comment:
What do you think? This is kind of weird, because we're using apps to lint dashboard. Maybe it does make sense to just provide a `npm run lint-dashboard` in apps?